pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -265,7 +265,8 @@ func (ghs *githubScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 				ghs.logger.Sugar().Errorf("error getting merged pull request count", zap.Error(err))
 			}
 
-			prPages := getNumPages(float64(100), float64(prOpenCount.Repository.PullRequests.TotalCount+prMergedCount.Repository.PullRequests.TotalCount))
+			totalPrCount := add(prOpenCount.Repository.PullRequests.TotalCount, prMergedCount.Repository.PullRequests.TotalCount)
```

Comment:
TotalCount will always be of type `int` correct?

Reply:
yes since add is of type `int` but it should never be a partial value anyways.  I guess should add be changed to float so it can be used everywhere else then just be casted to `int` for totalPrCount?

Reply:
yea, that or a generic